pkg/api/processapi/processapi_windows.go

```diff
@@ -9,6 +9,7 @@ type MsgCreateProcessEvent struct {
 	ParentProcessID   uint32
 	CreatingProcessID uint32
 	CreatingThreadID  uint32
+	UserLUID          uint64
```

Comment:
Hi @ExceptionalHandler sorry missed last windows dev, but I want to know is this how the field will be presented to users?  or is it internal to us only dev?

On linux loginuid https://tetragon.io/docs/reference/grpc-api/#process auid field is same but uid (effective uid) will change according to privileges, here it seems the opposite right? 

Feels more of an effective uid in Linux world... what are the other User IDs that we could report? just curious.